### DISCUSSION TITLE
Welcome to storify Discussions!

### DISCUSSION THREAD

**Discussion by gugadev** [gugadev/wc-stories#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gugadev** [comment]
Hi there! I'm retaking the Stencil refactor of Storify! Please leave here your most wanted features!

Here's mine:

- Complete refactor to StencilJS
- Add video support - [#22](https://github.com/gugadev/storify/issues/22)

**Reply by daviethedev** [reply]
Hey,
have you added the video support?
